CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Kubernetes in Rust : The contributing guide.
+
+Thank you for your interest in this project. In order to contribute to this project, please get familiar with the contributing rules and the git workflow.
+
+## Git workflow :
+
+![Git workflow](https://i.imgur.com/WMNLhY8.jpg)
+
+This project is organized within 2 repositories :
+- [The main repository](https://github.com/dev-sys-do/kudo)
+- [The development repository](https://github.com/do3-2021/kudo)
+
+The development repository is a fork of the main repository. All changes made to this repository are eventually going to get through a pull request of the upstream ( main ) repository.
+
+In the development repository, you will find 3 types of branches :
+- The main branch
+- Team's branch
+- Features ( or bugfix ) branches
+
+You must code in a feature branch. This feature branch will eventually end up in a PR to your team's branch. At this level, only your team's members are required to review. This PR is annotated with **PR 1** in the image above.
+
+After accepting multiples PRs by your team's members, your team's branch will get to a point where you feel it's ready to close an issue ( more on that later ). At this point, you must open a PR from you team's branch to the main branch ( **PR 2** ). This PR must be reviewed by at least 5 reviewers before getting marked as accepted.
+
+When all the teams are satisfied with the work in the main branch, and everybody agrees on the correctness of what's been done, a new PR, from this fork to the parent repo ( the main repo ) ( **PR 3** ), will be opened to be discussed with the parent's repo maintainer. 
+
+## How to commit
+
+First, let's list the basic rules :
+- A commit message has a title ( summary ), body and footer
+- The title should be 50 characters maximum
+- Capitalise the first letter of title, don't put a period ( . ) at the end
+- Put a blank line between the title, body and footer
+- Wrap the commit body at 72 characters
+- Try to write the commit body as **bullets** ( optional )
+- Use imperative in title ( **Add** instead of **Added**, for example )
+- Describe what and why, but not how ( the purpose, not the implementation )
+- Footer contains issue number, if applicable ( #78 )
```

Comment:
Is `#78`an example? What does adding `#78`mean? Does that mean the commit is related to issue #78? Does it fix it?

Reply:
Added a clearer explanation in the latest commit as well !